### ISSUE TITLE
Enable a PVector interfacea as in Processing Python Mode

### ISSUE THREAD

**Issue by berinhard** [berinhard/pyp5js#125]
This issue was created after #114 and the object is to have a `PVector` implementation as described by [Processing's docs](https://processing.org/reference/PVector.html). 

A possible approach is to add a `PVector` method to the `pyp5js/pyp5js.py` file to work a proxy for the `p5.Vector` implementation. Something similar to:

```python
def PVectorAdd(c1, c2):
    return _P5_INSTANCE.Vector.add(c1, c2)

def PVector(x, y):
    return _P5_INSTANCE.createVector(x, y)

PVector.add = PVectorAdd
```

**Comment by villares** [comment]
Trying my hand on a wrapper here: 
https://gist.github.com/villares/e639a34eef756beba2f79a55203cd51e

**Comment by villares** [comment]
Look how nice PVector tests!  https://github.com/jdf/processing.py/blob/master/testing/resources/test_pvector.py

**Comment by villares** [comment]
Hi @berinhard, as in Transcrypt we won't get the operator overloading, what do you think of a very thin helper layer for  p5.Vector using the PVector names?

Something like this:
```
from pyp5js import *

def setup():
    createCanvas(200, 200)
    background(160)

def draw():
    fill("blue")
    background(200)
    radius = sin(frameCount / 60) * 50 + 50
    
    v = PVector(100, 100)
    v.div(2)
    ellipse(v.x, v.y, radius, radius)
    v.add(30, 30)
    ellipse(v.x, v.y, radius, radius)
    w = PVector.add(v, PVector(50,0))
    ellipse(w.x, w.y, radius, radius)
    z = PVector.div(w, 2)
    ellipse(z.x, z.y, radius, radius)

def PVector(x=0, y=0, z=0):
    return createVector(x, y, z)

def __dist(a, b):
        return a.dist(b)
PVector.dist = __dist

def __add(a, *args):
    r = a.copy()
    return r.add(*args)
PVector.add = __add

def __div(v, f):
    r = v.copy()
    return r.div(f)
PVector.div = __div
```

**Comment by berinhard** [comment]
This set of aliases is extremely valid IMHO! Give it a try @villares and I hope it'll work =)

**Comment by berinhard** [comment]
@villares don't know if this can help you, but here's how we're setting attrs to isolate python functions such as `map`, `set` and `filter`: https://github.com/berinhard/pyp5js/blob/develop/pyp5js/python_functions.py

**Comment by berinhard** [comment]
Fixed by #137
